{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/make_artifacts.py

```diff
@@ -39,5 +39,137 @@ def build_artifacts(location: str, output_dir: str, append: bool, verbose: int):
         How noisy the logger should be.
 
     """
-    # Stubbed out pending further consideration of best location for this code.
-    pass
+    output_dir = Path(output_dir)
+    existing = ([output_dir / f'{sanitize_location(loc)}.hdf' for loc in project_globals.LOCATIONS]
```

Comment:
NIT: if the ternary clause extends over three lines, you might as well just use an if/else